kazel/generator.go

```diff
@@ -42,7 +42,7 @@ func (v *Vendorer) walkGenerated() error {
 		return nil
 	}
 	v.managedAttrs = append(v.managedAttrs, "openapi_targets", "vendor_targets")
-	paths, err := v.findOpenAPI(v.root)
+	paths, err := v.findOpenAPI(".")
```

Comment:
`SkippedPaths` uses relative paths, so the regexes won't match unless we use a relative path consistently. We use relative paths (i.e. `.`) for both `walk()` and `walkSource()`, so this was the odd one out.